src/workspaces/common/state/useWorkspaceStatePolling.ts

```diff
@@ -99,32 +102,34 @@ export const useWorkspaceStatePolling = (workspaces: Workspace[], status: Loaded
   }, [workspaces, status]); // adding the controller to deps causes a double fire of the effect
 };
 
-// we need a separate implementation of this, because using useWorkspaceStatePolling with a list
-// containing a single workspace in the WorkspaceContainer causes a rendering loop
-export const useSingleWorkspaceStatePolling = (workspace: Workspace) => {
```

Comment:
The previous rendering bug requiring a separate implementation for a single workspace vs multiple has been fixed. So to monitor a single workspace, we now just pass in an array with a single element.